lib/typings/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { MaterialUiPickersDate } from './date'
+import { isEqual, isAfter, endOfDay, setHours } from 'date-fns';
```

Comment:
change the file name to `.d.ts` and remove this import. I think this is redundant.

Reply:
@alitaheri I have removed date-fns imports, have no idea how they appeared there :) 
this file just a common interface for moment-utils and date-fns-utils